### ISSUE TITLE
Documenting request body

### ISSUE THREAD

**Issue by dzhus** [metosin/compojure-api#168]
With compojure-api-0.24.0, specifying documentation for body schema with
```clojure
:body [body (describe {:articles [Article]} "New content")]
```
for a route does not produce any documentation for body parameter in swagger spec.

**Comment by ikitommi** [comment]
hi. does the `s/named` work with that?

**Comment by vincentbernat** [comment]
Hey!

I run into the same problem: no description. Using `s/named` doesn't change anything:

```clojure
:body [log (s/named schema/LogEntry "log entry to append")]
```

Also, is it possible to give a name to this "parameter" instead of "Body24502"?

**Comment by ikitommi** [comment]
should be possible. will check this one out.

**Comment by ikitommi** [comment]
Sorry for taking so long.

Naming an anonymous body schema:

```clj
(POST "/some" []
  :body [body (s/schema-with-name {:kikka String} "Kikka")]
  (ok body))
```

Adding description to body schema fixed in `1.1.0-SNAPSHOT`:

```clj
(POST "/some" []
  :body [body (describe {:kikka String} "this is kikka")]
  (ok body))
```

Together:

```clj
(POST "/some" []
  :body [body (describe (s/schema-with-name {:kikka String} "Kikka") "this is kikka")]
  (ok body))
```

After implementing  [this](https://github.com/metosin/schema-tools/issues/11) feature, can add support for more declarative schemas.

**Comment by dzhus** [comment]
Thanks!
